spring-cloud-dataflow-container-registry/pom.xml

```diff
@@ -23,6 +23,12 @@
 		<dependency>
 			<groupId>com.amazonaws</groupId>
 			<artifactId>aws-java-sdk-ecr</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>commons-logging</groupId>
```

Comment:
Is this unrelated? I find it helpful to leave a self-review comment alerting the reviewer to these types of changes. Something like, 
```
**[UNRELATED]**: blah blah blah
```